packages/react-virtualizer/src/components/Virtualizer/renderVirtualizer.tsx

```diff
@@ -1,6 +1,3 @@
-/** @jsxRuntime automatic */
-/** @jsxImportSource @fluentui/react-jsx-runtime */
```

Comment:
this will break slots. you need to add following

```
/** @jsxRuntime classic */
/** @jsx createElement */
import { createElement } from '@fluentui/react-jsx-runtime';
```